maintainers/maintainer-list.nix

```
@@ -15389,6 +15389,12 @@
     githubId = 3958212;
     name = "Tom Sorlie";
   };
+  schinmai-akamai = {
+    email = "schinmai@akamai.com";
+    github = "schinmai-akamai";
+    gitubId = 70169773;
```

>  error: The option `lib.maintainers.schinmai-akamai.gitubId' does not exist. Definition values:
---
Looks like a spelling mistake, this should fix it!
```suggestion
    githubId = 70169773;
```

---

Great catch!. fixed it